tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteConsumerGroupsTest.java

```diff
@@ -337,7 +335,6 @@ private Map<String, Object> composeConfigs(ClusterInstance cluster, String group
         configs.put(KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
         configs.put(VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
         configs.put(GROUP_PROTOCOL_CONFIG, groupProtocol);
-        configs.put(PARTITION_ASSIGNMENT_STRATEGY_CONFIG, RangeAssignor.class.getName());
```

Comment:
just to double check, even though we're removing this prop, I expect the test will still use the range assignor if under classic protocol just because range is the first assignor in the default value of the config. Correct?